x-pack/legacy/plugins/siem/public/components/open_timeline/index.test.tsx

```diff
@@ -9,21 +9,41 @@ import { mount } from 'enzyme';
 import { MockedProvider } from 'react-apollo/test-utils';
 import React from 'react';
 import { ThemeProvider } from 'styled-components';
+import { getOr } from 'lodash/fp';
 
 import { wait } from '../../lib/helpers';
 import { TestProviderWithoutDragAndDrop, apolloClient } from '../../mock/test_providers';
 import { mockOpenTimelineQueryResults } from '../../mock/timeline_results';
 import { DEFAULT_SEARCH_RESULTS_PER_PAGE } from '../../pages/timelines/timelines_page';
 
-import { StatefulOpenTimeline } from '.';
 import { NotePreviews } from './note_previews';
 import { OPEN_TIMELINE_CLASS_NAME } from './helpers';
-
+import { StatefulOpenTimeline } from '.';
+import { useGetAllTimeline, getAllTimeline } from '../../containers/timeline/all';
 jest.mock('../../lib/kibana');
+jest.mock('../../containers/timeline/all', () => {
+  const originalModule = jest.requireActual('../../containers/timeline/all');
+  return {
+    useGetAllTimeline: jest.fn(),
+    getAllTimeline: originalModule.getAllTimeline,
+  };
+});
 
 describe('StatefulOpenTimeline', () => {
   const theme = () => ({ eui: euiDarkVars, darkMode: true });
   const title = 'All Timelines / Open Timelines';
+  beforeEach(() => {
+    ((useGetAllTimeline as unknown) as jest.Mock).mockReturnValue({
+      fetchAllTimeline: jest.fn(),
+      timelines: getAllTimeline(
+        '',
+        getOr([], 'getAllTimeline.timeline', mockOpenTimelineQueryResults[0].result.data)
```

Comment:
nit -> I think now we can avoid the getOR by `mockOpenTimelineQueryResults[0].result.data?. getAllTimeline?.timeline ??[]`